client/etcd/client.go

```diff
@@ -347,6 +354,10 @@ func (c *csclient) sanitizeOptions(opts kv.OverrideOptions) (kv.OverrideOptions,
 		opts = opts.SetEnvironment(c.opts.Env())
 	}
 
+	if opts.WatchWithRevision() == 0 {
```

Comment:
should be `if opts.WatchWithRevision() != 0`?

Reply:
My intention here was that if the override doesn't specify a start revision it will inherit the base config's. This does however mean that if the base config sets `WatchWithRevision`, the override can't disable it. I think that's fine since that's what we do for environment and namespace as well.